libcxx/utils/ci/buildkite-pipeline.yml

```diff
@@ -101,20 +101,6 @@ environment_definitions:
 
 
 steps:
-- label: Bootstrapping build
```

Comment:
I think we can remove the `linux_agent` definition above now!

Reply:
Great!

Setting up the Mac OS X bots should be really easy too! Let me know if you want some help.